src/foam/u2/view/ScrollTableView.js

```diff
@@ -190,7 +227,40 @@
       class: 'Boolean',
       name: 'isInit'
     },
-    'tableWrapper_'
+    'tableWrapper_',
+
+    // Navigation props
```

Comment:
Please document the next three properties.

Reply:
Done